--- php-5.3.23.ori/ext/standard/info.c	2013-03-18 07:39:47.000000000 +0100
+++ php-5.3.23/ext/standard/info.c	2013-03-18 07:38:58.000000000 +0100
@@ -691,6 +691,10 @@
 
 		if (!sapi_module.phpinfo_as_text) {
 			php_printf("<h1 class=\"p\">PHP Version %s</h1>\n", PHP_VERSION);
+#ifdef __x86_64__
+#define ENTROPY_CH_THIS_ARCH "x86_64"
+#endif
+			php_printf("<h2 class=\"p\"><a href=\"http://php-osx.liip.ch/\">php-osx.liip.ch</a> by <a style=\"text-decoration: none; background-color:inherit;\" href='http://www.liip.ch'>Liip</a> (originally developed by <a style=\"text-decoration: none; background-color:inherit;\" href='http://www.local.ch'>www.local.ch</a>)</h2>\n");
 		} else {
 			php_info_print_table_row(2, "PHP Version", PHP_VERSION);
 		}
